.github/workflows/publish-rust.yml

```diff
@@ -20,15 +20,6 @@ on:
           - patch
           - minor
           - major
-          - rc
-          - beta
-          - alpha
-          - release
-          - version
-      version:
-        description: Version (used with level "version")
-        required: false
-        type: string
```

Comment:
You can bring lines 23-31 back since it works with the updated way to run semver-checks.